concrete/src/File/Image/BasicThumbnailer.php

```diff
@@ -158,42 +160,44 @@ public function getThumbnail($obj, $maxWidth, $maxHeight, $crop = false)
                 $fr = $obj->getFileResource();
                 $fID = $obj->getFileID();
                 $filename = md5(implode(':', array($fID, $maxWidth, $maxHeight, $crop, $fr->getTimestamp())))
-                  . '.' . $fh->getExtension($fr->getPath());
+                  .'.'.$fh->getExtension($fr->getPath());
             } catch (\Exception $e) {
                 $filename = '';
             }
         } else {
             $filename = md5(implode(':', array($obj, $maxWidth, $maxHeight, $crop, filemtime($obj))))
-                . '.' . $fh->getExtension($obj);
+                .'.'.$fh->getExtension($obj);
         }
 
-        $abspath = '/cache/' . $filename;
+        $abspath = '/cache/'.$filename;
 
-        $src = $configuration->getPublicURLToFile($abspath);
+        if ($obj instanceof File) {
+            $customThumb = new CustomThumbnail($maxWidth, $maxHeight, $abspath, false);
```

Comment:
Same thing here as the other `new ...` call, are we sure we want/need this dependency to live here?